index.html

```diff
@@ -162,17 +162,26 @@
             For details on what kinds of behaviour are not tolerated and consequences for violating these rules, we refer to the <a href="http://rubyberlin.github.io/code-of-conduct">Berlin Code of Conduct</a>.
           </p>
         </div>
+        <div>
+          <h2>Privacy Policy and leagal</h2>
+          <p>
+            This site is run by
+            <a href="http://bitcrowd.net" title="bitcrowd.net software development for startups">@bitcrowd</a>.
+            You can find all leagal information <a href="http://bitcrowd.net" title="bitcrowd.net software development legal">here</a>.
```

Comment:
Perhaps make 'all legal information here' the link, instead of just 'here' — it’s more descriptive